src/controllers/update/update.confirmation.controller.ts

```diff
@@ -0,0 +1,38 @@
+import { NextFunction, Request, Response } from "express";
```

Comment:
just a small thing - should we be prefixing our controller names with `update.` if it's already within the update folder?